### DISCUSSION TITLE
Weird hitching when mashing random in song select

### DISCUSSION THREAD

**Discussion by bytenekos** [ppy/osu#25792] [Q&A]
### Type

Performance

### Bug description

If I mash random in song select, sometimes it stops playing the sound effect that plays when hitting the random button, then after a short time it "catches up" and plays the sound effects that it missed (I think that's what's happening anyway)

### Screenshots or videos

Here's a clip showcasing this:

https://github.com/ppy/osu/assets/66184829/3c481702-bd84-4076-ac7b-b80c0bc5bfb6

### Version

2023.1130.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/13691516/database.log)
[input.log](https://github.com/ppy/osu/files/13691517/input.log)
[network.log](https://github.com/ppy/osu/files/13691518/network.log)
[performance.log](https://github.com/ppy/osu/files/13691519/performance.log)
[runtime.log](https://github.com/ppy/osu/files/13691520/runtime.log)
[updater.log](https://github.com/ppy/osu/files/13691521/updater.log)

**Comment by bytenekos** [comment]
I have no idea why, but seems to not be an issue after a restart..? So now I don't know if it's a real issue, or if it was a one-off kind of thing. During the play session that I had when I had the bug, I could reproduce it "consistently". Just opened osu again and I can't get it to happen now

**Comment by peppy** [comment]
Might have been recalculating difficulties or something.

**Comment by bytenekos** [comment]
What would cause a difficulty recalculation? Just so I know if this ever happens again

**Comment by peppy** [comment]
No idea, but I can't see anything wrong in the video.
